docs/content-guideline-edge-cases.md

```diff
@@ -3,23 +3,36 @@ The boundary between police brutality, abuse of power, and other issues with the
 ## Police speaking words that would incite violence, but not actively violent during this video
 
 - [Cops at Seattle protest brief "don't kill them but hit them hard"](https://www.reddit.com/r/PublicFreakout/comments/gwr1gh/cops_at_a_seattle_protest/)
-- Decision - TBD
+
+### Decision
+
+TBD
+
 
 ## Selective enforcement, abuse of power, not necessarily excessive force
 
-- [Cops arresting only the black individuals in a line](https://www.reddit.com/r/PublicFreakout/comments/gwm2mf/police_using_selective_enforcement_on_protestors/)
-- [Cops giving warning & preferential treatment to Proud Boys and other white supremacists](ttps://www.buzzfeednews.com/article/davidmack/salem-oregon-cop-warn-white-armed-men-playing-favorites)
-- Decision - egregious selective enforcement will be in scope.
+- [Cops arresting only the black individuals in a line](https://www.reddit.com/r/PublicFreakout/comments/gwm2mf/police_using_selective_enforcement_on_protestors/) ✔️ Added in [#260](https://github.com/2020PB/police-brutality/pull/260)
+- [Cops giving warning & preferential treatment to Proud Boys and other white supremacists](ttps://www.buzzfeednews.com/article/davidmack/salem-oregon-cop-warn-white-armed-men-playing-favorites) ✔️ Added in [#626](https://github.com/2020PB/police-brutality/pull/626)
+
+### Decision
+
+Egregious selective enforcement will be in scope.
+
 
 ## Agitation and/or incitement
 
-- [Fargo police chief agitated crowd in plain clothes by carrying a beer can and cursing at police.](https://www.grandforksherald.com/opinion/6520779-Port-Fargo-Deputy-Chief-sends-apology-email-after-he-was-seen-at-George-Floyd-protest-with-a-beer-can-cursing-at-law-enforcement)
-- Decision - agitation and incitement to riot will be in scope.
+- [Fargo police chief agitated crowd in plain clothes by carrying a beer can and cursing at police.](https://www.grandforksherald.com/opinion/6520779-Port-Fargo-Deputy-Chief-sends-apology-email-after-he-was-seen-at-George-Floyd-protest-with-a-beer-can-cursing-at-law-enforcement) ✔️ Added in [#620](https://github.com/2020PB/police-brutality/pull/620)
+
+### Decision
+
+Agitation and incitement to riot will be in scope.pply the `incitement` tag to these reports.
+
 
 ## No direct visual evidence
 
 - [Protester suffered from complications that lead to her death from tear gas](https://twitter.com/ColumbusGov/status/1268295227921641480)

```

Comment:
I couldn't find this case in the current reports, if anyone knows what the history/decision was with this one then it would be great to add it?

Reply:
https://www.thelantern.com/2020/06/columbus-looks-into-death-of-ohio-state-grad-who-attended-downtown-protests/

Looks like it's still under review (autopsy could take weeks) but I think the popular narrative is that pepper spray was a factor. That's not saying it's true enough to include it

Reply:
Which I think supports it being in this category

Reply:
Gotcha, and as far as I can see we haven't added it to [Ohio.md](https://github.com/2020PB/police-brutality/blob/master/reports/Ohio.md#columbus) yet?

Reply:
I believe you're correct but I'm taking this to mean we should still exclude it?